account_invoice_start_end_dates/tests/test_invoice_start_end_dates.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import time
+
+from odoo import fields
+from odoo.tests.common import SavepointCase
+from odoo.tools import float_compare
+
+
+class TestInvoiceStartEndDates(SavepointCase):
```

Comment:
```suggestion
@tagged('-at_install', 'post_install')
class TestInvoiceStartEndDates(SavepointCase):
```

Reply:
When running the test on the latest version of the Odoo code the following error is raise:
ValueError: Expected singleton: account.journal()
This is because the Chart of Account is not yet configured when this module is installed, this is done in a post install hook.
Therfore this test needs to be run in the post-install step